astropy/modeling/functional_models.py

```diff
@@ -854,6 +854,22 @@ def fit_deriv(x, amplitude, x_0, fwhm):
         d_fwhm = 2 * amplitude * d_amplitude / fwhm * (1 - d_amplitude)
         return [d_amplitude, d_x_0, d_fwhm]
 
+    def bounding_box(self, factor=25):
+        """Tuple defining the default ``bounding_box`` limits,
+        ``(x_low, x_high)``.
+
+        Parameters
+        ----------
+        factor : float
+            The multiple of FWHM used to define the limits.
+            Similar to `Gaussian1D`.
```

Comment:
@pllim I wonder if this line is a bit confusing or at least unnecessary.
Also is the factor meaningful or chosen arbitrarily?